### PR TITLE
Stanislav Hryhorskiy: Added homeworks 8-10

### PR DESCRIPTION
**Pull request by Stanislav-Hryhorskyi**
I'm waiting for the code review

### CODE REVIEW

File: Stanislav_Hryhorskiy/HW10/hw_10_practical_task_1.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# *** Practical task 1 ***
+# Create a polygon class and a rectangle class
+# that inherits from the polygon class and finds
+# the square of rectangle.
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW10/hw_10_practical_task_1.py:5]: good job

File: Stanislav_Hryhorskiy/HW10/hw_10_practical_task_2.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# *** Practical task 2 ***
+# Create a class Human, everyone has a name,
+# create a method in the class that displays
+# a welcome message to each person. Create a class
+# method in the class that returns information
+# that it is a species of "Homosapiens". And in the
+# class create a static method that returns an
+# arbitrary message.
+
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW10/hw_10_practical_task_2.py:10]: good job

File: Stanislav_Hryhorskiy/HW10/hw_10_practical_task_3.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
+# *** Practical task 3 ***
+# Create an employee class. Each employee has characteristics such as name and salary.
+# The class should have a counter that calculates the total number of employees,
+# as well as a method that prints the total number of employees and a method that
+# displays information about each employee in particular, namely the name and salary.
+# In addition to creating a class, display information about the base classes from
+# which the employee class is inherited (__base__), the class namespace (__dict__),
+# the class name (__name__), the module name in which the class is defined (__module__),
+# the documentation bar (__doc__).
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW10/hw_10_practical_task_3.py:10]: good job

File: Stanislav_Hryhorskiy/HW10/hw_10_practical_tasks_codewars.py [hunk 4]
```diff
@@ -0,0 +1,126 @@
+# I. Ball super ball
+# Create a class Ball. Ball objects should accept
+# one argument for "ball type" when instantiated.
+# If no arguments are given, ball objects should
+# instantiate with a "ball type" of "regular."
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW10/hw_10_practical_tasks_codewars.py:6]: good job

File: Stanislav_Hryhorskiy/HW10/time_to_practice_10.py [hunk 5]
```diff
@@ -0,0 +1,32 @@
+# Create a Car class with the attributes name, kind, model
+# and methods of start and stop, which indicate information
+# that the car started or stopped accordingly.
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW10/time_to_practice_10.py:4]: good job

File: Stanislav_Hryhorskiy/HW11/hw_11_practical_task_1.py [hunk 6]
```diff
@@ -0,0 +1,41 @@
+# *** Task 1 ***
+# Write a program that prompts the user to enter
+# their age, and then displays a message stating
+# whether the age is even or odd. The program must
+# provide the ability to enter a negative number,
+# and in this case generate an exception.
+# The master code should call a function
+# that processes the information entered.
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW11/hw_11_practical_task_1.py:9]: excellent job

File: Stanislav_Hryhorskiy/HW11/hw_11_practical_task_2.py [hunk 7]
```diff
@@ -0,0 +1,42 @@
+# *** Task 2 ***
+# Write a program that analyzes the entered number and,
+# depending on the number, gives the day of the week that
+# corresponds to this number (1 is Monday, 2 is Tuesday, etc.).
+# Take into account cases of entering numbers from 8 and more,
+# as well as cases of entering nonnumerical data.
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW11/hw_11_practical_task_2.py:7]: good job

File: Stanislav_Hryhorskiy/HW11/time_to_practice_11.py [hunk 8]
```diff
@@ -0,0 +1,40 @@
+# *** Task 1 ***
+# Write a program that prompts the user
+# to enter an integer and determines whether
+# the number is even or odd, taking into account
+# the case of entering incorrect data.
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW11/time_to_practice_11.py:6]: good job

File: Stanislav_Hryhorskiy/HW8/hw8_practical_task_2/task_2.py [hunk 9]
```diff
@@ -7,6 +7,30 @@
 # Minimum length 6 characters.
 # Maximum length 16 characters.
 
-password = input("Enter your password: ")
+import re
 
-# TODO using REGEX
\ No newline at end of file
+MIN_LENGTH, MAX_LENGTH = 6, 16
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW8/hw8_practical_task_2/task_2.py:13]: good job

File: Stanislav_Hryhorskiy/HW8/time_to_practice_8/calculator.py [hunk 10]
```diff
@@ -0,0 +1,14 @@
+# *** Time to practice 8 ***
+# Create a module with functions of addition, subtraction,
+# multiplication, division. And in another module - calculator.py, it is
+# necessary to ask the user what action he wants to perform and
+# with what numbers and perform the necessary actions.
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW8/time_to_practice_8/calculator.py:6]: good job

File: Stanislav_Hryhorskiy/HW8/time_to_practice_8/functions.py [hunk 11]
```diff
@@ -0,0 +1,75 @@
+def get_add_num(first_number, second_number: float) -> str:
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW8/time_to_practice_8/functions.py:1]: excellent implementation

File: Stanislav_Hryhorskiy/HW9/OWM.py [hunk 12]
```diff
@@ -0,0 +1,26 @@
+from pyowm import OWM
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW9/OWM.py:1]: good job

File: Stanislav_Hryhorskiy/HW9/hw9_practical_task_1.py [hunk 13]
```diff
@@ -0,0 +1,50 @@
...
+# of 1 to 100 and asks the user to guess that number in 10 tries.
+# The program reads the numbers entered by the user and prompts the user
+# whether the guessed number is greater or less than the number entered
+# by the user. The game must continue until the user has used 10 attempts
+# and guessed the number. If the user guessed the number, the program prints
+# a congratulatory message, and if 10 attempts have been exhausted and the user
+# did not have time to guess the number, then the corresponding message
+# is displayed. (to perform the task, you need to import the random module,
+# and from it the randint() function)
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW9/hw9_practical_task_1.py:12]: good job

File: Stanislav_Hryhorskiy/HW9/hw9_practical_task_2_(Rect_pygame).py [hunk 14]
```diff
@@ -0,0 +1,55 @@
+# *** Practical task 2 ***
+# You need to change the code in the 'hw9_practical_task_2_(Rect_pygame).py'
+# program so that the rectangle does not extend beyond the program window.
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW9/hw9_practical_task_2_(Rect_pygame).py:4]: good job

File: Stanislav_Hryhorskiy/HW9/hw9_practical_task_3.py [hunk 15]
```diff
@@ -0,0 +1,69 @@
+# *** Practical task 3 ***
+# You need to combine two programs OWM.py and Tk_OWM.py
+# into one working program.
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW9/hw9_practical_task_3.py:4]: good job

File: Stanislav_Hryhorskiy/HW10/hw_10_practical_task_1.py (hunk 1 above)
Comment (Stanislav-Hryhorskyi) [reply Stanislav_Hryhorskiy/HW10/hw_10_practical_task_1.py:5]: thanks!

File: Stanislav_Hryhorskiy/HW10/hw_10_practical_task_2.py (hunk 2 above)
Comment (Stanislav-Hryhorskyi) [reply Stanislav_Hryhorskiy/HW10/hw_10_practical_task_2.py:10]: thanks!

File: Stanislav_Hryhorskiy/HW10/hw_10_practical_tasks_codewars.py (hunk 4 above)
Comment (Stanislav-Hryhorskyi) [reply Stanislav_Hryhorskiy/HW10/hw_10_practical_tasks_codewars.py:6]: thanks!

File: Stanislav_Hryhorskiy/HW8/time_to_practice_8/functions.py (hunk 11 above)
Comment (Stanislav-Hryhorskyi) [reply Stanislav_Hryhorskiy/HW8/time_to_practice_8/functions.py:1]: thanks!

File: Stanislav_Hryhorskiy/HW9/hw9_practical_task_1.py (hunk 13 above)
Comment (Stanislav-Hryhorskyi) [reply Stanislav_Hryhorskiy/HW9/hw9_practical_task_1.py:12]: thanks!

File: Stanislav_Hryhorskiy/HW10/hw_10_practical_task_3.py (hunk 3 above)
Comment (Stanislav-Hryhorskyi) [reply Stanislav_Hryhorskiy/HW10/hw_10_practical_task_3.py:10]: thanks!

File: Stanislav_Hryhorskiy/HW10/time_to_practice_10.py (hunk 5 above)
Comment (Stanislav-Hryhorskyi) [reply Stanislav_Hryhorskiy/HW10/time_to_practice_10.py:4]: thanks!

File: Stanislav_Hryhorskiy/HW11/hw_11_practical_task_1.py (hunk 6 above)
Comment (Stanislav-Hryhorskyi) [reply Stanislav_Hryhorskiy/HW11/hw_11_practical_task_1.py:9]: thanks!

File: Stanislav_Hryhorskiy/HW11/hw_11_practical_task_2.py (hunk 7 above)
Comment (Stanislav-Hryhorskyi) [reply Stanislav_Hryhorskiy/HW11/hw_11_practical_task_2.py:7]: thanks!

**Pull request merged**